МСГ начало connects to #83

GIdroponik54 opened the pull request:
1) добавил класс
2) минимальная реализация без предобуславливания
3)   Seidel на     GZ заменил, потому что Алина уже так написала, если нужно могу обратно перевести. Совпадение не обязательно, но как-то не по феншую что-ли.  Ну и соответсвенно я за нее добавил поэтому мой проект не будет компилируется (case Type.GZ:
-                    return new src.Solver.GZ(); ), без него компилилось. 
  ![image](https://cloud.githubusercontent.com/assets/17140800/13264331/d0b409a4-da97-11e5-8e08-a1ca3ae7f006.png) вот пруфик

Commit:
добавила новый класс решателя ГауссаЗейделя

gidroponik54 committed:
1) Реализовал МСГ без пред. (еще не тестился)
2) в Ilovere добавил кэйсы для моего метода и GZ поэтому без слияния ее ветки не будет работать (поздно заметили эту штуку)

Swipes13 commented:
Лучше наоборот - поменяй GZ на GaussSeidel. Всё-таки так пишется фамилия великого Зейделя ;)

Swipes13 commented:
И не могло из-за этого свалится, гуи вообще пофиг, что там за энум. Не верю пруфику :)
в этой ветке вообще нет файла GZ.cs потому она и валится )

ghost commented:
case Type.GZ:

return new src.Solver.GZ();
вот из-за этого валится ГЗ в ее реквесте запилен, этот файл во втором реквесте запилен, я же написал

gidroponik54 committed:
GaussSeidel добавил великого

Swipes13 commented:
Понял, тогда просто удаляй у себя этот код, это нужно добавить в ее пр )

Swipes13 commented on toop-project/toop-project/src/Solver/MSG.cs:
```diff
@@ -0,0 +1,68 @@
+            }
+        }
+
+        public string Name
```
Тоже самое, что в пр #116

Swipes13 commented on toop-project/toop-project/src/Solver/MSG.cs:
```diff
@@ -0,0 +1,68 @@
+{
+    public class MSG : ISolver
+    {
+
```
Удаляем вайтспейсы

Swipes13 commented on toop-project/toop-project/src/Solver/MSG.cs:
```diff
@@ -0,0 +1,68 @@
+                z = rOld;
+
+                /////////////
+                //Start
```
Лишний комментарий 

Всем и так понятно,  что это старт )

Stepdan committed:
Дописал диагональный формат (не тестирован).

Commit:
в Isolver добавила в switch GS

gidroponik54 committed:
1)  Короче теперь компилится.
2) добавил странную функцию в классе, которая теперь выполняет роль возращения имени.

Commit:
new Type GetType()

gidroponik54 committed:
если  яправильно понял удалить вайтпасы, то именно это я и сделал)

Commit:
get name убрала
